contributor/03-contributor-ethos.md

```diff
@@ -0,0 +1,64 @@
+# Draft comments
+
+ Outline for this section
+
+     Contributors by definition work in teams that they are not familiar with. This section explains how to avoid common pitfalls when crossing team boundaries.
+
+ ## Article
```

Comment:
I would suggestion to use `#` for draft comment and # for the article and then `##(###, etc.)` for the (sub-)sections as this might make it easier to read during the drafting process.
```suggestion
# Article
```